### DISCUSSION TITLE
Regarding JSON scada login issue

### DISCUSSION THREAD

**Discussion by Nitesh1794** [riclolsen/json-scada#113] [Q&A]
Hey riclosen !
Please check login credentials. It is showimng Login error, invalid credentials !
While using : username=admin password=jsonscada

**Comment by riclolsen** [comment]
Hi Nitesh,

Are you running it on Windows? Which exact version?

Please check if the service "JSON_SCADA_server_realtime_auth" is up and running.

**Comment by Nitesh1794** [comment]
Yes I am using windows 10 and JSON SCADA version is 0.28 alpha. and  "JSON_SCADA_server_realtime_auth" is up and running.
But login it is displaying that "Login error, invalid credentials!". Please check.

**Comment by riclolsen** [comment]
Enter mongodb database using Compass.

You must have this user entry:

![image](https://github.com/riclolsen/json-scada/assets/18125204/c392105f-e4a1-45fe-9b11-62c15a0d52f1)

If not there was a problem initializing the database. Try uninstalling and reinstalling.

**Reply by Nitesh1794** [reply]
Thanks, working now.
